cassandra/io/eventletreactor.py

```diff
@@ -81,6 +94,7 @@ def service_timeouts(cls):
 
     def __init__(self, *args, **kwargs):
         Connection.__init__(self, *args, **kwargs)
+        self.uses_legacy_ssl_options = self.ssl_options and not self.ssl_context
```

Comment:
This is much easier than doing an instance check :)

Reply:
I must admit that this helps my brain :D